### PR TITLE
ARS 5.0 Overlay

### PR DESCRIPTION
**Pull request by ssayed118**
- [x] Check the list of inputs updated in [RHEL 8 overlay](https://github.com/CMSgov/cms-ars-5.0-red-hat-enterprise-linux-8-stig-overlay/tree/main)
- [x] Run overlay using all system categorizations:
  - [x] High-HVA
  - [x] High
  - [x] Moderate-HVA
  - [x] Moderate
  - [x] Low-HVA
  - [x] Low
- [x] README

### CODE REVIEW

File: controls/overlay.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  end
+
+  control 'SV-214799' do
+    tag nist: ["CM-14"]
+  end
+  
+  ## SC-45 (1) incorporates withdrawn control AU-8 (1)
+  control 'SV-204603' do
+    tag nist: ["SC-45 (1)"]
+  end
```
Comment (karikarshivani) [inline controls/overlay.rb:86]: Looks like an `end` may be missing

File: inspec.yml [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+maintainer: CMS SAF team
+copyright: .
+copyright_email: saf@cms.hhs.gov
+license: Apache-2.0
+summary: "CMS ARS 5.0 Overlay InSpec Validation Profile for RHEL7 STIG"
+version: 1.3.1
+inspec_version: ">= 4.0"
+
+depends:
+- name: redhat-enterprise-linux-7-stig-baseline
```
Comment (karikarshivani) [inline inspec.yml]: Is the indentation off here?

File: inspec.yml [hunk 3]
```diff
@@ -0,0 +1,259 @@
+name: cms-ars-5.0-red-hat-enterprise-linux-7-stig-overlay
+title: cms-ars-5.0-red-hat-enterprise-linux-7-stig-overlay
+maintainer: CMS SAF team
+copyright: .
+copyright_email: saf@cms.hhs.gov
+license: Apache-2.0
+summary: "CMS ARS 5.0 Overlay InSpec Validation Profile for RHEL7 STIG"
+version: 1.3.1
```
Comment (karikarshivani) [inline inspec.yml]: @ejaronne I forget if we align this with STIG or ARS or it's independent

File: inspec.yml [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+  # SV-204393, SV-204394, SV-204395, SV-204397, SV-204426, SV-204431, SV-204447, SV-204448, SV-204497, SV-204578, SV-204580, SV-204595, SV-204603, SV-204631, SV-204632, SV-204633
+  - name: org_name 
+    desc: "Name of the organization running this profile"
+    type: String
+    value: "CMS"
+    profile: redhat-enterprise-linux-7-stig-baseline
+
+  # SV-204513, SV-204514
+  - name: storage_volume
+    desc: Amount of audit storage volume
```
Comment (karikarshivani) [inline inspec.yml:31]: This should probably be okay (and we can test it) but I usually see `desc` values set as strings

File: inspec.yml [hunk 5]
```diff
@@ -0,0 +1,259 @@
+name: cms-ars-5.0-red-hat-enterprise-linux-7-stig-overlay
+title: cms-ars-5.0-red-hat-enterprise-linux-7-stig-overlay
```
Comment (karikarshivani) [inline inspec.yml:2]: Delete this duplicate repo https://github.com/mitre/cms-ars-5.0-red-hat-enterprise-linux-7-stig-overlay

File: inspec.yml (hunk 2 above)
Comment (ssayed118) [reply inspec.yml]: it looks fine on my end

File: inspec.yml (hunk 4 above)
Comment (ssayed118) [reply inspec.yml:31]: I can take out the 'desc' part if needed

File: inspec.yml (hunk 2 above)
Comment (karikarshivani) [reply inspec.yml]: The indentation was actually off, but it's fixed now

File: inspec.yml (hunk 3 above)
Comment (karikarshivani) [reply inspec.yml]: I've updated the version to 3.10.0 to match STIG

Comment (karikarshivani) [reply inspec.yml]: Discussed and agreed upon

**Review by karikarshivani — APPROVED** [review]
LGTM!

**Pull request merged**